src/main/kotlin/no/nav/syfo/aktivitetskrav/AktivitetskravService.kt

```diff
@@ -62,6 +63,15 @@ class AktivitetskravService(
         aktivitetskrav: Aktivitetskrav,
         aktivitetskravVurdering: AktivitetskravVurdering,
     ) {
+        if (aktivitetskravVurdering.status == AktivitetskravStatus.FORHANDSVARSEL) {
+            throw ConflictException("Kan ikke sette FORHANDSVARSEL her, bruk aktivitetskravVarselService.sendForhandsvarsel")
+        }
+        val currentVurdering = aktivitetskrav.vurderinger.firstOrNull()
+        if (currentVurdering?.isFinal() == true) {
+            throw ConflictException("Aktivitetskravet har allerede en avsluttende vurdering")
+        }
```

Comment:
Her har vi `isFinal` property på `AktivitetskravStatus` vi kan bruke😊 Er det en grunn til at `LUKKET` og `STANS` er utelatt fra `finalStatuses` i `AktivitetskravStatus`? 
```suggestion
        if (currentVurdering?.status?.isFinal == true) {
            throw ConflictException("Aktivitetskravet har allerede en avsluttende vurdering")
        }
```

Reply:
Ja, det vil virke det, men jeg synes det er bedre å ha en isFinal-metode i vurdering-klassen som delegerer til statusen.

Reply:
Men du har rett om det med LUKKET og STANS, kan vel bruke isFinal-property'en på statusen ser det ut som 👍

(tror dette har endret seg litt underveis)